index.html

```diff
@@ -4,7 +4,403 @@
+            </div>
+          </li>
+
+          <li class="benefits__item">
+            <span class="benefits__background"></span>
+            <div class="benefits__wrapper">
+              <div class="benefits__heading">
+                <svg class="benefits__img">
+                  <use
+                    xlink:href="./img/sprite-black-icon.svg#icon-events-black"
+                  ></use>
+                </svg>
+                <h3 class="benefits__subtitle">Live Digital Events</h3>
+              </div>
+              <p class="benefits__description">
+                Online Plus Talks featuring conversations with industry leaders
+                across media, marketing, branding industries, virtual networking
+                opportunities and access to video archives.
+              </p>
+
+              <div class="benefits__view">
+                <a class="benefits__link" href="#">View a sample</a>
+                <button class="benefits__button button" type="button">
+                  <img
+                    class="benefits__img-btn"
+                    src="./img/union.svg"
+                    width="12"
+                    height="12"
+                    alt="View a sample"
+                  />
+                </button>
+              </div>
+            </div>
+          </li>
+
+          <li class="benefits__item">
+            <span class="benefits__background"></span>
+            <div class="benefits__wrapper">
+              <div class="benefits__heading">
+                <svg class="benefits__img">
+                  <use
+                    xlink:href="./img/sprite-black-icon.svg#icon-stories-black"
+                  ></use>
+                </svg>
+                <h3 class="benefits__subtitle">Unlimited stories</h3>
+              </div>
+              <p class="benefits__description">
+                Full access to all content published on Digiday. Non-members are
+                limited to 3 articles each month.
+              </p>
+
+              <div class="benefits__view">
+                <a class="benefits__link" href="#">View a sample</a>
+                <button class="benefits__button button" type="button">
+                  <img
+                    class="benefits__img-btn"
+                    src="./img/union.svg"
+                    width="12"
+                    height="12"
+                    alt="View a sample"
+                  />
+                </button>
+              </div>
+            </div>
+          </li>
+        </ul>
+      </section>
+
+      <section class="faq">
+        <h2 class="faq__subtitle subtitle">Frequently Asked Questions</h2>
+        <ul class="faq__list">
+          <li class="faq__item">
+            <input type="checkbox" id="faq__checkbox-1" name="faq__group" />
+            <label class="faq__heading" for="faq__checkbox-1"
+              >What type of memberships are offered?
+            </label>
+            <section class="faq__content">
```

Comment:
`section` tag is supposed to serve to some larger content blocks, here it might be a `p` or a `div`.